README.md

```diff
@@ -25,6 +25,66 @@ We follow [Google's coding style guides](https://google.github.io/styleguide/).
 
 # Development
 
+## Setting up
+
+### Log in `gcloud` with a Service Account
+
+Instead of using your personal credential to log in, it's recommended
+to use a Service Account instead.
+
+A new Service Account and proper permissions can be created using the
+following commands. `PROJECT-ID` is the (non-numeric) name of your
```

Comment:
nit: maybe change "name" to "identifier", since "project name" is actually another separate field.